packages/components/RadioGroup/src/legacy/RadioButton.win32.tsx

```diff
@@ -0,0 +1,217 @@
+/** @jsxRuntime classic */
+/** @jsx withSlots */
+'use strict';
+import * as React from 'react';
+import { View, I18nManager } from 'react-native';
+
+import { filterViewProps } from '@fluentui-react-native/adapters';
+import {
+  useAsPressable,
+  //useOnPressWithFocus,
+  useViewCommandFocus,
+  KeyPressEvent,
+  useKeyDownProps,
+} from '@fluentui-react-native/interactive-hooks';
+import { Text } from '@fluentui-react-native/text';
+import { foregroundColorTokens, textTokens, borderTokens, backgroundColorTokens, getPaletteFromTheme } from '@fluentui-react-native/tokens';
+import type { ISlots } from '@uifabricshared/foundation-composable';
+import { withSlots } from '@uifabricshared/foundation-composable';
+import type { IUseComposeStyling } from '@uifabricshared/foundation-compose';
+import { compose } from '@uifabricshared/foundation-compose';
+import { mergeSettings } from '@uifabricshared/foundation-settings';
+
+import { settings, radioButtonSelectActionLabel } from './RadioButton.settings';
+import type { IRadioButtonType, IRadioButtonProps, IRadioButtonSlotProps, IRadioButtonRenderData } from './RadioButton.types';
+import { radioButtonName } from './RadioButton.types';
+import { RadioGroupContext } from './RadioGroup';
+
+enum DirectionalArrowKeys {
+  ArrowUp = 'ArrowUp',
+  ArrowDown = 'ArrowDown',
+  ArrowRight = 'ArrowRight',
+  ArrowLeft = 'ArrowLeft',
+}
+
+export const RadioButton = compose<IRadioButtonType>({
+  displayName: radioButtonName,
```

Comment:
What of this file is a fork from the base file?

Reply:
The win32 specific parts added on to the contents of the base file are primarily lines 77-139.